components/templates/NewsPage/client.js

```diff
@@ -1,24 +1,16 @@
 "use client";
 import PropTypes from "prop-types";
 import { useTranslation } from "react-i18next";
-import {
-  getSiteString,
-  useCustomBreadcrumbs,
-  makeReleaseFeature,
-} from "@/lib/utils";
-import { useRelease } from "@/lib/api/noirlabReleases";
+import { useCustomBreadcrumbs } from "@/lib/utils";
 import Breadcrumbs from "@/page/Breadcrumbs";
 import NewsHero from "./NewsHero";
 import NewsArticle from "./NewsArticle";
 import NewsList from "@/dynamic/NewsList";
 import NewsAside from "@/components/page/Aside/patterns/Media";
 import * as Styled from "./styles";
+import { getReleaseHero } from "@/lib/api/noirlab";
 
 export default function NewsPage({ data }) {
```

Comment:
instead of mixing the data in clientside, a new server component parent of this page handles it. More content on this page could be moved serverside by shifting it to the server component page, but not a very high priority and would be better done in a server tree expansion ticket